README.md

```diff
@@ -1,23 +1,24 @@
-# BROADEN.IO Backend
+[![eslint: airbnb](https://img.shields.io/badge/eslint-airbnb-%23fd5c63.svg)](https://www.npmjs.com/package/eslint-config-airbnb)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-broaden.io
 
-## Table of Contents
+# Backend
 
-# BROADEN.IO
-
-Backend API Server that supports the Single Page application that allows students to achieve measurable feedback by giving them tools for diagnostic, self-assessment through rubrics created by educators. This is project for Make School. You can reach us at through our contact below.
-
-[FrontEnd Repro](https://github.com/Jeffchiucp/Rubrics-Frontend) -- GitHub: https://github.com/Jeffchiucp/Rubrics-Frontend
+# 
+Broaden.io is a web app that helps fledgling developers assess tech skills. Students can use this to get measurable feedback through self-assessment . Users can use this platform to get measurable feedback through self-assessment. You can reach us at through our contact below. 
```

Comment:
I prefer the older description, but if you want to keep this one, I recommend removing the line "Students can use this to get measurable feedback through self-assessment ." because the next line mentioning users is the same thing. In addition, if you want to change the description, I think it would be fair to have someone else's take on it to make it a deciding vote, maybe @tlambrou ?